analytics/api/src/main/java/mil/nga/giat/geowave/analytic/GeometryDataSetGenerator.java

```diff
@@ -490,92 +481,94 @@ private SimpleFeature createFeatureWithGeometry(
 
 	}
 
-	public static void main(
-			final String args[] )
-			throws Exception {
-		final Options allOptions = new Options();
-		DataStoreCommandLineOptions.applyOptions(allOptions);
-		final Option typeNameOption = new Option(
-				"typename",
-				true,
-				"a name for the feature type (required)");
-		typeNameOption.setRequired(true);
-		allOptions.addOption(typeNameOption);
-		CommandLine commandLine = new BasicParser().parse(
-				allOptions,
-				args);
-
-		final CommandLineResult<DataStoreCommandLineOptions> dataStoreOption = DataStoreCommandLineOptions.parseOptions(
-				allOptions,
-				commandLine);
-		if (dataStoreOption.isCommandLineChange()) {
-			commandLine = dataStoreOption.getCommandLine();
-		}
-		else {
-			throw new ParseException(
-					"Unable to parse data store from command line");
-		}
-		final DataStore dataStore = dataStoreOption.getResult().createStore();
-		final String typeName = commandLine.getOptionValue("typename");
-		final GeometryDataSetGenerator dataGenerator = new GeometryDataSetGenerator(
-				new FeatureCentroidDistanceFn(),
-				getBuilder(typeName));
-		dataGenerator.writeToGeoWave(
-				dataStore,
-				dataGenerator.generatePointSet(
-						0.2,
-						0.2,
-						5,
-						5000,
-						new double[] {
-							-100,
-							-45
-						},
-						new double[] {
-							-90,
-							-35
-						}));
-		dataGenerator.writeToGeoWave(
-				dataStore,
-				dataGenerator.generatePointSet(
-						0.2,
-						0.2,
-						7,
-						5000,
-						new double[] {
-							0,
-							0
-						},
-						new double[] {
-							10,
-							10
-						}));
-		dataGenerator.writeToGeoWave(
-				dataStore,
-				dataGenerator.addRandomNoisePoints(
-						dataGenerator.generatePointSet(
-								0.2,
-								0.2,
-								6,
-								5000,
-								new double[] {
-									65,
-									35
-								},
-								new double[] {
-									75,
-									45
-								}),
-						6000,
-						new double[] {
-							-90,
-							-90
-						},
-						new double[] {
-							90,
-							90
-						}));
-	}
+	// public static void main(
```

Comment:
do we use this anymore?  If not, can we just delete it?  (if so, we probably have a problem in the first place here, so I'll assume we don't)